Quaver.Shared/Database/Playlists/PlaylistManager.cs

```diff
@@ -25,6 +28,11 @@ public static class PlaylistManager
         /// </summary>
         public static List<Playlist> Playlists { get; private set; } = new List<Playlist>();
 
+        /// <summary>
+        ///     Tasks list
+        /// </summary>
+        public static List<Task> Tasks { get; private set; } = new List<Task>();
+
```

Comment:
This shouldn't be a property if it's only being used in one method and nowhere else.